run.py

```diff
@@ -52,17 +56,37 @@ def current_request():
     # TODO: change this
     engine.session = session
 
-    # fix-up the postgres schema:
+    alembic_cfg = Config("alembic.ini")
+    alembic_cfg.attributes['configure_logger'] = False
+
+    logger.info(f"Creating database with uri '{db_connection}'")
```

Comment:
Is there a reason we've moved away from the SDC logging standards?

Reply:
Not sure that log line was already there. Is this against sdc logging standards?

Reply:
https://collaborate2.ons.gov.uk/confluence/display/SDC/Logging

Reply:
That was something @jcox-dev added in the party service refactor